### ISSUE TITLE
Sonar code coverage are not generated

### ISSUE THREAD

**Issue by clemens-tolboom** [molgenis/molgenis-service-armadillo#704]
### How to Reproduce

- Visit https://github.com/molgenis/molgenis-service-armadillo
- Check the shields ![image](https://github.com/molgenis/molgenis-service-armadillo/assets/371014/78e2e9df-f94d-4675-b397-30313ed82c37)

### Expected behavior

- Coverage should have a non zero percentage
- Quality gate should not fail

### Observed behavior

- Coverage: 0%
- Quality gate: failed

**Comment by clemens-tolboom** [comment]
PR #695 seems not to fix anything ... checking the CICD logs shows

> Task :jacocoTestReport SKIPPED

Trying to run locally

```
./gradlew :jacocoTestReport
# Does nothing
```

and

```
./gradlew jacocoTestReport
# runs test
```
ends with
```
...
RExecutorServiceImplTest > testLoadResource() PASSED
#
BUILD SUCCESSFUL in 36s
```

**Comment by marikaris** [comment]
Fixed in #734
